File: codeforces/263A/cpp/263a.cpp
#include <iostream>
#include <cmath>
using namespace std;

const int c_size = 5;

int main()
{
    int cell[c_size][c_size];
    int row = 0, col = 0;
    for (int i = 0; i < c_size; i++) {
        for (int j = 0; j < c_size; j++) {
            cin >> cell[i][j];
            if (cell[i][j] == 1) {
                row = i;
                col = j;
            }
        }
    }

    int mid = c_size / 2;
    cout << fabs(row - mid) + fabs(col - mid) << endl;
    return 0;
}

File: codeforces/263A/go/263a.go
package main

import "fmt"

func calc_abs(n int) int {
    if n >= 0 {
        return n
    }
    return -n
}

func main() {
    const c_size int = 5
    var cell[c_size][c_size] int

    row := 0
    col := 0
    for i := 0; i < c_size; i++ {
        for j := 0; j < c_size; j++ {
            if j == c_size - 1 {
                fmt.Scanf("%d\n", &cell[i][j])
            } else {
                fmt.Scanf("%d", &cell[i][j])
            }
            if cell[i][j] == 1 {
                row = i
                col = j
            }
        }
    }

    mid := c_size / 2;
    fmt.Println(calc_abs(row - mid) + calc_abs(col - mid))
}
